package.json

```diff
@@ -11,6 +11,10 @@
     "build": "yarn workspaces foreach --all --topological --verbose run build",
     "publint": "yarn workspaces foreach --all --topological --verbose run publint",
     "publish": "yarn workspaces foreach --all --no-private --topological --verbose npm publish --tolerate-republish && changeset tag",
+    "version:npm": "changeset version",
+    "version:jsr": "yarn workspaces foreach --all --no-private --parallel run version:jsr",
+    "version:get": "jq -r .version $INIT_CWD/package.json",
```

Comment:
Each workspace then calls this global script to get the version from it's package.json

I couldn't find an easy way to do this with `yarn` 🤔